cpp/src/glm/qn/qn_util.cuh

```diff
@@ -179,7 +179,7 @@ inline int lbfgs_search_dir(const LBFGSParam<T> &param, int *n_vec,
     // "L-BFGS-B Fortran subroutines for large-scale bound constrained
     // optimization" Ciyou Zhu, Richard H. Byrd, Peihuang Lu and Jorge Nocedal
     // (1994).
-    CUML_LOG_INFO("L-BFGS WARNING: skipping update step ys=%f, yy=%f", ys, yy);
+    CUML_LOG_WARN("L-BFGS WARNING: skipping update step ys=%f, yy=%f", ys, yy);
```

Comment:
This change will make the message to appear more, not less. I think the change you're looking for would be to use either debug or trace levels?

Reply:
Ahh, I was confused which direction it increased in. Thanks!